MetaMask/metamask-extension
# Add SecuX Wallet at QR-based option.

## Explanation

## Screenshots/Screencaps

### Before

<img width="377" alt="截圖 2023-04-26 下午4 40 47" src="https://user-images.githubusercontent.com/46667542/234520347-7be72e78-e99d-4e89-b5f9-90db0be404ee.png">

### After

<img width="380" alt="截圖 2023-04-26 下午4 41 49" src="https://user-images.githubusercontent.com/46667542/234520301-a8ea3b87-113a-4efe-90f2-1c05a7cda1bc.png">

## Manual Testing Steps

- Go to My Accounts
- Click Connect hardware wallet
- Click QR-based and Continue button
- See the SecuX option in the list below

## Pre-merge author checklist


## Pre-merge reviewer checklist
